javascript-operators-and-expressions/main.js

```diff
@@ -38,5 +38,5 @@ console.log('value of isSparta', isSparta);
 console.log('type of isSparta', typeof isSparta);
 
 var motto = fullName;
-console.log('value of motto', motto, 'is the GOAT');
+console.log('value of motto' + motto + 'is the GOAT');
```

Comment:
You should be doing `motto += "is the GOAT."`, and then you can just log `motto`